angularSTEP/src/app/login/login.component.ts

```diff
@@ -18,7 +18,7 @@ export class LoginComponent implements OnInit {
   //and there is no user in the database linked to the account
   //it will delete the user (in case user refreshes page before
   //setting a username but after signing in with Google
-  constructor(public fAuth: AngularFireAuth, public router: Router) {}
+  constructor(private fAuth: AngularFireAuth, private router: Router) {}
```

Comment:
It's really nice to limit visibility as much as possible up front! Thanks for switching a bunch of these away from public